MBINCompiler/Models/NMSTemplate.cs

```diff
@@ -126,6 +126,17 @@ public static NMSTemplate DeserializeBinaryTemplate(BinaryReader reader, string
                         }
 
                         break;
+                    case "Object":

```

Comment:
The code is there for lists, but not for single instances.

I could change the type to be NMSTemplate instead of Object and rename the switch cases, though.